managed/services/server/updater_test.go

```diff
@@ -245,14 +251,29 @@ func TestUpdater(t *testing.T) {
 		}
 	})
 
+	t.Run("TestSortedVersionList", func(t *testing.T) {
+		versions := version.DockerVersionsInfo{
+			{Version: *version.MustParse("3.0.0")},
+			{Version: *version.MustParse("3.1.0")},
+			{Version: *version.MustParse("3.0.1")},
+			{Version: *version.MustParse("3.0.0-rc")},
+		}
+
+		sort.Sort(versions)
+		assert.Equal(t, "3.0.0-rc", versions[0].Version.String())
+		assert.Equal(t, "3.0.0", versions[1].Version.String())
+		assert.Equal(t, "3.0.1", versions[2].Version.String())
+		assert.Equal(t, "3.1.0", versions[3].Version.String())
+	})
+
 	t.Run("TestLatest", func(t *testing.T) {
 		version.Version = "2.41.0"
```

Comment:
This test will fail for now since the changes to version service have not been deployed.